microlith/Dockerfile

```diff
@@ -254,6 +274,8 @@ ENV GF_INSTALL_PLUGINS=""
 COPY prometheus/ /etc/prometheus/
 COPY alertmanager/ /etc/alertmanager/
 COPY jaeger/ /etc/jaeger/
+COPY promtail/ /etc/promtail/
+ENV CMOS_LOGS=/logs
```

Comment:
NITPICKING: possibly `CMOS_LOGS_ROOT` or a more descriptive name?

Reply:
Also, if we're making this user-customisable, should we update the `tee` in `run.sh` to use that variable instead? Currently it's hardcoded to write to `/logs/"${EXE_NAME}".log`.

Reply:
Yeah I did but then the rebase overwrote it!